var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -44,15 +42,46 @@ class Kokkos(Package):
     version('2.03.00', 'f205d659d4304747759fabfba32d43c3')
     version('2.02.15', 'de41e38f452a50bb03363c519fe20769')
     version('2.02.07', 'd5baeea70109249f7dca763074ffb202')
+    version('develop', git='https://github.com/kokkos/kokkos',
+            branch='develop')
```

Comment:
Why that change?

Reply:
An excellent question! I'm not sure that I recall, so I just reverted it below.